app/scripts/controllers/permissions/specifications.js

```diff
@@ -278,6 +279,16 @@ function validateCaveatAccounts(accounts, getInternalAccounts) {
   });
 }
 
+export const unrestrictedEthSigningMethods = Object.freeze([
```

Comment:
Also, let's move it before (or after?) the declaration of `unrestrictedMethods`.

```suggestion
/**
 * Unrestricted methods for Ethereum, see {@link unrestrictedMethods} for more details.
 */
export const unrestrictedEthSigningMethods = Object.freeze([
```